rosetta-source/src/main/resources/cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/expectations.json

```diff
@@ -7,15 +7,15 @@
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex19-double-no-touch-option.xml","excludedPaths":0,"externalPaths":71,"outstandingMappings":43,"validationFailures":4,"qualificationExpectation":{"success":false,"qualifyResults":[],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex21-avg-rate-option-parametric-plus-rate-observation.xml","excludedPaths":0,"externalPaths":67,"outstandingMappings":18,"validationFailures":4,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_VanillaOption","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex21-avg-rate-option-parametric-plus-rate-observation.xml","excludedPaths":0,"externalPaths":67,"outstandingMappings":18,"validationFailures":5,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_VanillaOption","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex23-straddle.xml","excludedPaths":0,"externalPaths":69,"outstandingMappings":56,"validationFailures":1,"qualificationExpectation":{"success":true,"qualifyResults":[],"qualifiableObjectCount":0}},
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex24-delta-hedge.xml","excludedPaths":0,"externalPaths":66,"outstandingMappings":53,"validationFailures":1,"qualificationExpectation":{"success":true,"qualifyResults":[],"qualifiableObjectCount":0}},
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex25-option-strategyComponentIdentifier.xml","excludedPaths":0,"externalPaths":97,"outstandingMappings":64,"validationFailures":1,"qualificationExpectation":{"success":true,"qualifyResults":[],"qualifiableObjectCount":0}},
-{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex26-fxswap-multiple-USIs.xml","excludedPaths":0,"externalPaths":64,"outstandingMappings":15,"validationFailures":8,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Swap","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/fx-derivatives/fx-ex26-fxswap-multiple-USIs.xml","excludedPaths":0,"externalPaths":64,"outstandingMappings":15,"validationFailures":12,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Swap","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
```

Comment:
FX sample does not specify rate source, so required field (x2) `primaryFxSpotRateSource` is not set